src/modules/sponsor/setEdition.ts

```diff
@@ -0,0 +1,38 @@
+import chalk from 'chalk'
+import * as R from 'ramda'
+import { Sponsor } from '../../clients/sponsor'
+import * as conf from '../../conf'
+import { UserCancelledError } from '../../errors'
+import log from '../../logger'
+import { default as switchAccount } from '../auth/switch'
+import { promptConfirm } from '../prompts'
+import { switchToPreviousAccount } from '../utils'
+import { getIOContext, options } from './utils'
+
+const promptChangeToSponsorAccount = async (sponsorAccount: string) => {
+  const proceed = await promptConfirm(
+    `Do you wish to log into the sponsor account ${sponsorAccount}?`
+  )
+  if (!proceed) {
+    throw new UserCancelledError()
+  }
+  await switchAccount(sponsorAccount, {})
+}
+
+export default async (edition: string) => {
+  const previousConf = conf.getAll()
+  const previousAccount = conf.getAccount()
```

Comment:
**nit**: I guess you can use `previousConf.account`

Reply:
Sure